components/Carousel/Carousel.js

```diff
@@ -1,4 +1,73 @@
-class Carousel {
+class CarouselItem {
+    constructor(element) {
+        this.element = element;
+    }
+    select() {
+        this.element.classList.add("Carousel_item-selected");
+    }
+
+    deselect() {
+        this.element.classList.remove("Carousel__item-selected");
+    }
```

Comment:
You may want another method that allows for you to get the data index for the carousel!

Reply:
Scratch my previous comment, I realized how you were implementing a tracker for your carousel below.